### DISCUSSION TITLE
Next version

### DISCUSSION THREAD

**Discussion by rwbaer** [DillonHammill/CytoExploreR#176] [Q&A] [answered]
I know you have been working on a new version for some time now.   I thought I'd ask how its coming and if you have any sort of time projection on it being ready for prime time or early testing?

**Comment by DillonHammill** [comment, marked as answer]
CytoExploreR version 2.0.0 is almost ready, I still have some configuration issues to fix before it will be publically available.

The new version requires updates to other cytoverse packages which haven't been merged yet. As soon as that is sorted out I will release it for beta testing. This could happen in the next couple of weeks.

**Comment by rwbaer** [comment]
Great.  Version 1.1 has been very useful to me, and I'm excited to see what you have done in the new version.

**Comment by rwbaer** [comment]
Thought enough time might have gone by to check on the status here.  Did the cytoverse merges ever get successfully completed?
